test/fixtures/01-basic/config.ts

```diff
@@ -0,0 +1,31 @@
+import { readFileSync } from 'fs';
+import type { AbstractSqlModel } from '@balena/abstract-sql-compiler';
+import type { ConfigLoader } from '../../../src/server-glue/module';
+import * as sbvrUtils from '../../../src/sbvr-api/sbvr-utils';
+
+const generateAbstractSqlModel = (seModelPath: string): AbstractSqlModel => {
+	const seModel = readFileSync(seModelPath, 'utf8');
+	const lfModel = sbvrUtils.generateLfModel(seModel);
+	return sbvrUtils.generateAbstractSqlModel(lfModel);
+};
+
+const apiRoot = 'university';
+const modelName = 'university';
+const abstractSql = generateAbstractSqlModel(__dirname + '/university.sbvr');
+
+export default {
+	models: [
+		{
+			modelName,
+			abstractSql,
+			apiRoot,
+		},
+	],
+	users: [
+		{
+			username: 'guest',
+			password: ' ',
+			permissions: ['resource.all'],
+		},
+	],
+} as ConfigLoader.Config;
```

Comment:
There's no reason to manually parse/compile the model, you can just pass the sbvr file location directly in:

```suggestion
import type { ConfigLoader } from '../../../src/server-glue/module';

const apiRoot = 'university';
const modelName = 'university';
const modelFile = __dirname + '/university.sbvr';

export default {
	models: [
		{
			modelName,
			modelFile,
			apiRoot,
		},
	],
	users: [
		{
			username: 'guest',
			password: ' ',
			permissions: ['resource.all'],
		},
	],
} as ConfigLoader.Config;
```

Reply:
Thanks, will do so.